tests/assertions/is_not_substring_of.zsh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env zunit
+
+@test 'not a substring of' {
+    assert 'foo' is_not_substring_of 'yellow'
+}
```

Comment:
Can you also add a test for the failure state, and assert that `$output` is `1` and check the error message please? Again, see the `same_as` example above.